src/break/Break.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+import { body } from '../theme/fonts';
+
+import { Break } from './';
+
+const style = {
+  fontFamily: body,
+}
+
+storiesOf('Break', module)
+  .add('Break', () => <Break fontColor='white'
```

Comment:
I'm not sold on the name of this component. A break reminds me of `<hr/>`. Isn't this just a specific type of Section? Like a gradient Section?